src/hooks/spoofing/write_player_gamer_data_node.cpp

```diff
@@ -3,36 +3,37 @@
 
 namespace big
 {
-	void hooks::write_player_gamer_data_node(rage::netObject* player, CPlayerGamerDataNode* node)
-	{
-		g_hooking->get_original<write_player_gamer_data_node>()(player, node);
+    void hooks::write_player_gamer_data_node(rage::netObject* player, CPlayerGamerDataNode* node)
+    {
+        g_hooking->get_original<write_player_gamer_data_node>()(player, node);
 
-		if (g.spoofing.spoof_crew_data)
-		{
-			node->m_clan_data.m_clan_member_id = 1;
-			node->m_clan_data.m_clan_id = 41564112;
-			node->m_clan_data.m_clan_id_2 = 41564112;
-			node->m_clan_data.m_clan_color = 420;
-			node->m_clan_data.m_clan_member_count = 15;
-			node->m_clan_data.m_clan_created_time = 420;
+        // move these variables to a separate function
+        const bool spoof_rockstar_id = g.spoofing.spoof_rockstar_id;
+        const bool spoof_rockstar_dev = spoof_rockstar_id && g.spoofing.spoof_rockstar_dev;
+        const bool spoof_rockstar_qa = spoof_rockstar_id && g.spoofing.spoof_rockstar_qa;
```

Comment:
No need to move these outside of their respective scopes, the compiler will optimise this as it'll deem fit.

Reply:
I have reverted it, should be fixed now.